src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/General.BrowseObject.xaml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright, Microsoft Corporation, All rights reserved.-->
+<Rule
+  Name="ConfigurationGeneralBrowseObject"
+  DisplayName="General"
+  PageTemplate="generic"
+  Description="General"
+  OverrideMode= "Replace"
+  xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="General" DisplayName="General" Description="General" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile" Label="Configuration" HasConfigurationCondition="False" />
+  </Rule.DataSource>
+
+  <StringProperty Name="ApplicationIcon" DisplayName="Application Icon" />
+  <DynamicEnumProperty Name="TargetFrameworkMoniker" DisplayName="Target Framework Moniker" EnumProvider="TargetFrameworkMoniker" />
+  <StringProperty Name="AssemblyName" DisplayName="Assembly Name" />
+  <StringProperty Name="Name" Visible="False" />
+  <StringProperty Name="RootNamespace" DisplayName="Root namespace" />
+  <StringProperty Name="DefaultNamespace" DisplayName="Default namespace">
+    <StringProperty.DataSource>
+      <DataSource PersistedName="RootNamespace" Persistence="ProjectFile" Label="Configuration" HasConfigurationCondition="false" />
+    </StringProperty.DataSource>
+  </StringProperty>
+  <StringProperty Name="TargetFramework" Visible="False">
+    <StringProperty.DataSource>
+      <DataSource Persistence="ProjectFile" PersistedName="TargetFrameworkMoniker" />
+    </StringProperty.DataSource>
+  </StringProperty>
+  <StringProperty Name="OutputName" Visible="False" />
+  <DynamicEnumProperty Name="OutputType" DisplayName="Output Type" EnumProvider="OutputTypeEnumProvider" />
+  <EnumProperty Name="OutputTypeEx" DisplayName="Output Type">
+    <EnumValue Name="dll" DisplayName="0" />
+    <EnumValue Name="exe" DisplayName="1" />
+    <EnumValue Name="winexe" DisplayName="2" />
+    <EnumValue Name="WinMDObj" DisplayName="3" />
```

Comment:
I'll lower case it for consistency but it doesn't seem to matter. Looks like CPS does a case-insensitive comparison. No for OutputTypeEx the enum provider doesn't play any role. It just comes from this enum.